docs/logs/using.asciidoc

```diff
@@ -72,6 +72,19 @@ image::logs/images/logs-usage-streaming-indicator.png[Logs streaming indicator]
 
 Historical data offers infinite scrolling.
 
+[float]
+[[logs-highlight]]
+== Highlight a phrase in the logs stream
```

Comment:
Suggest making this a head 3 so it is at the same level as as the "Inspect a log" heading.